packages/core/stories/form-field/form-field.stories.tsx

```diff
@@ -511,3 +516,253 @@ export const WithInputWithAdornments: ComponentStory<typeof FormField> = (
     </FlowLayout>
   );
 };
+
+export const GroupedWithLabelTop: ComponentStory<typeof FormField> = (
+  props
+) => {
+  return (
+    <StackLayout>
+      <FormField {...props}>
+        <FormLabel>Form Field label left</FormLabel>
+        <Input defaultValue="Value" />
+        <FormHelperText>Helper text</FormHelperText>
+      </FormField>
+      <FormField {...props}>
+        <FormLabel>
+          Form Field label that's extra long. Showing that labels wrap around to
+          the line.
+        </FormLabel>
+        <Input defaultValue="Primary Input value" />
+      </FormField>
+      <FormField {...props}>
+        <FormLabel>Form Field label</FormLabel>
+        <Input defaultValue="Value" />
+        <FormHelperText>Helper text</FormHelperText>
+      </FormField>
+      <FormField {...props}>
+        <FormLabel>Form Field label</FormLabel>
+        <Input defaultValue="Primary Input value" />
+      </FormField>
+    </StackLayout>
+  );
+};
+
+export const GroupedWithLabelLeft: ComponentStory<typeof FormField> = (
+  props
+) => {
+  const groupedProps = { labelPlacement: "left" } as Partial<FormFieldProps>;
+  return (
+    <StackLayout>
+      <FormField {...groupedProps} {...props}>
+        <FormLabel>Form Field label left</FormLabel>
+        <Input defaultValue="Value" />
+        <FormHelperText>Helper text</FormHelperText>
+      </FormField>
+      <FormField {...groupedProps} {...props}>
+        <FormLabel>
+          Form Field label that's extra long. Showing that labels wrap around to
+          the line.
+        </FormLabel>
+        <Input defaultValue="Primary Input value" />
+      </FormField>
+      <FormField {...groupedProps} {...props}>
+        <FormLabel>Form Field label</FormLabel>
+        <Input defaultValue="Value" />
+        <FormHelperText>Helper text</FormHelperText>
+      </FormField>
+      <FormField {...groupedProps} {...props}>
+        <FormLabel>Form Field label</FormLabel>
+        <Input defaultValue="Primary Input value" />
+      </FormField>
+    </StackLayout>
+  );
+};
+
+export const GroupedWithLabelRight: ComponentStory<typeof FormField> = (
+  props
+) => {
+  const groupedProps = { labelPlacement: "right" as FormFieldLabelPlacement };
```

Comment:
Dislike the use of `as`.  Is it even needed?

Reply:
same, it wouldn't let me put it as it said string cant be applied to FormFieldLabelPlacement type. will try to adjust

Reply:
other than making an enum which doesn't feel right for this or putting the whole obj as `Partial<FormFieldProps>` I can't think of an alternative, always requires casting it from a string.. am i missing something obvious?

Reply:
```
import { type FormFieldLabelPlacement } from 'somewhere';

const groupedProps:{labelPlacement: FormFieldLabelPlacement} = { labelPlacement: "right"  };
```

maybe?